### ISSUE TITLE
Linux Link Bitpay Card

### ISSUE THREAD

**Issue by Joykiller** [bitpay/copay#6443]
Takes you to the main page log in, once logged in asks you to click button on website to link card however does nothing. Console output was giving CORS issue.

**Comment by JDonadio** [comment]
@Joykiller, are you building BitPay? 404 in most cases is shown when you are trying to make a http request from localhost.

**Comment by Joykiller** [comment]
nope, just downloaded the zip compiled linux wallet and went to add card, works fine on android and windows.
